launcher/screens/Guide/Dashboard/Footer/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {
+	Image,
+	ImageBackground,
+	ImageStyle,
+	View,
+	ViewStyle,
+} from 'react-native';
+import { DimensionState, dimensionState, Text } from '@metacraft/ui';
+import UnderRealmButton from 'components/Marketplace/Button';
+import { useDefaultHoveredStyle } from 'components/Marketplace/Button/shared';
+import { useSnapshot } from 'utils/hook';
+import resources from 'utils/resources';
+import { iStyles } from 'utils/styles';
+
+const Footer: React.FC<Record<string, unknown>> = () => {
+	const { windowSize } = useSnapshot<DimensionState>(dimensionState);
+	const width = Math.min(windowSize.width, iStyles.contentContainer.maxWidth);
+	const backgroundContainer = {
+		width,
+		height: (width * 385) / 1555,
+		alignItems: 'center',
+		justifyContent: 'center',
+	} as ViewStyle;
+
+	const imageSize = {
+		width: '50%',
+		aspectRatio: 884 / 222,
+	} as ImageStyle;
+	return (
+		<ImageBackground
+			source={resources.guide.footerMainBackground}
+			style={[iStyles.wideContainer, backgroundContainer]}
+		>
+			<View style={imageSize}>
+				<ImageBackground
+					source={resources.guide.footerContentBackground}
+					style={{ flex: 1, alignItems: 'center', paddingTop: 20 }}
```

Comment:
Avoid using incline style if there is more than 2 fields, use `StyleSheet`  instead